invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/footer.less

```diff
@@ -48,4 +48,4 @@
  }
  .spacer-long{
   margin-top: 20px;
-}
\ No newline at end of file
+}
```

Comment:
This is for me not a bugfix, it is of the type style. It does not change the meaning of the code.